client/src/components/ThemeText/ThemeText.css

```diff
@@ -0,0 +1,13 @@
+.headerText {
+  font-size: 40px;
+  color: white;
+  font-weight: bold;
+  font-family: "Roboto", sans-serif;
+}
+
+.subtitleText {
+  font-size: 18px;
```

Comment:
See comment from the `CredentialsButton.css` file regarding the use of rem instead of px

Reply:
Fixed: change to rem